Audacia.Typescript/Audacia.Typescript.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>3.1.0.0</Version>
+    <Version>3.1.0</Version>
```

Comment:
Doesn't look like a 3.1.0 has ever been published? In the 2.5 years this has been on Audacia.Public, it has never been downloaded 😆

Reply:
Yeah, this PR will increment the version to 3.1.0 :) My reasoning is that the current version (3.0.19197.41345) has four parts to it, and the versioning task will not handle that, so it needs to be incremented to a higher version. I could have gone with 3.0.19198, but I thought I'd take the opportunity to move to a more sensible version number.

It may well be that the versioning task blows up trying to parse the current version, but we'll see.

Reply:
Makes sense, thanks for explaining 👍